docs/contributing/javascript_guidelines.rst

```diff
@@ -47,10 +47,9 @@ you need to make to the codebase.
 Changing the linter configuration
 ---------------------------------
 
-Under the hood, the tasks use the `JavaScript Code Style <http://jscs.info/>`_ library.
+To edit ignored files, add an entry to the ``.eslintignore`` file.
```

Comment:
I know this was there before and you're just updating the docs (thank you!), but I don't think this is worth documenting.